tests/longevity/longevity_test.go

```diff
@@ -454,6 +479,28 @@ func populateIntervals() {
 	triggerInterval[BackupRestartNode][6] = 5 * baseInterval
 	triggerInterval[BackupRestartNode][5] = 6 * baseInterval
 
+	triggerInterval[AppIncreaseDecreaseScaleFactor][10] = 1 * baseInterval
+	triggerInterval[AppIncreaseDecreaseScaleFactor][9] = 2 * baseInterval
+	triggerInterval[AppIncreaseDecreaseScaleFactor][8] = 3 * baseInterval
+	triggerInterval[AppIncreaseDecreaseScaleFactor][7] = 4 * baseInterval
+	triggerInterval[AppIncreaseDecreaseScaleFactor][6] = 5 * baseInterval
+	triggerInterval[AppIncreaseDecreaseScaleFactor][5] = 6 * baseInterval
+	triggerInterval[AppIncreaseDecreaseScaleFactor][4] = 7 * baseInterval
+	triggerInterval[AppIncreaseDecreaseScaleFactor][3] = 8 * baseInterval
+	triggerInterval[AppIncreaseDecreaseScaleFactor][2] = 9 * baseInterval
```

Comment:
I think we need to create a different configmap for scaling based namespace wise